### DISCUSSION TITLE
[Comment] Anexo 1. "intentadas" o "tiradas"?

### DISCUSSION THREAD

**Discussion by PaulaMorBla14** [PaulaMorBla14/TFG-Baloncesto#9] [Q&A]
¿¿Qué escribir: "intentadas" o "tiradas"??

"Intentadas" me parece que sean las no anotadas (y no la suma de ambas, anotas y no anotadas)
"Tiradas" me parece poco formal para un trabajo (o mejor "lanzadas" ?)
"Realizadas" me da la sensación de que se han encestados

O mejor buscar algún sinónimo (?)

https://github.com/PaulaMorBla14/TFG-Baloncesto/blob/a507088506598503009b9669520982b77af8399a/Project_v1.Rmd#L141-L150

**Comment by solmos** [comment]
Yo usaría "intentados", y "tiros" en lugar de "canastas":

* Tiros libres intentados
* Tiros de 2 intentados
* Tiros de 3 intentados
